meson.build

```diff
@@ -0,0 +1,5 @@
+project('pgroonga', 'c', version: '4.0.2')
+
+subdir('pgrn-pgxs')
+subdir('pgroonga-headers')
+subdir('pgroonga-check')
```

Comment:
Why do we need to create these subdirectories?

Reply:
I just wanted to build each module using separate `meson.build`.
Without directory boundaries, the `meson.build` filenames would conflict, so we split them into individual folders.

Reply:
I think that we don't need separate `meson.build`. Our `meson.build` will not be so complex like `CMakeListst.txt`.

Reply:
Thanks, I understood it. I will fix the above way!